### PR TITLE
Add footer component

### PR DESCRIPTION
**Pull request by IshappaN**
This PR adds a new footer component to the application.

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/glucerna/components/footercopyRight/footercopyRight.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    <div class="logo_img">
+      <h4>${footer.text}</h4>
+
+      <div>
+        <sly data-sly-list.footerItems="${footer.fields}">
+          <img
+            class="footer_icons"
+            src="${footerItems.fileReference}"
+            width="25px"
+            height="25px"
```
Comment (SwaroopCodilar) [inline ui.apps/src/main/content/jcr_root/apps/glucerna/components/footercopyRight/footercopyRight.html]: Dont write inline css.

File: ui.apps/src/main/content/jcr_root/apps/glucerna/components/footercopyRight/_cq_dialog/.content.xml [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                                <field
+                                    jcr:primaryType="nt:unstructured"
+                                    sling:resourceType="granite/ui/components/coral/foundation/container"
+                                    name="./fields">
+                                    <items jcr:primaryType="nt:unstructured">
+                                        <icons
+                                            jcr:primaryType="nt:unstructured"
+                                            sling:resourceType="cq/gui/components/authoring/dialog/fileupload"
+                                            fieldLabel="Upload logo here"
+                                            fileReferenceParameter="./fileReference"
```
Comment (SwaroopCodilar) [inline ui.apps/src/main/content/jcr_root/apps/glucerna/components/footercopyRight/_cq_dialog/.content.xml]: provide root path

File: ui.apps/src/main/content/jcr_root/apps/glucerna/components/footercopyRight/footercopyRight.html (hunk 1 above)
Comment (IshappaN) [reply ui.apps/src/main/content/jcr_root/apps/glucerna/components/footercopyRight/footercopyRight.html]: okay I will change it

File: ui.apps/src/main/content/jcr_root/apps/glucerna/components/footercopyRight/_cq_dialog/.content.xml [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+                    <logoTab
+                        jcr:primaryType="nt:unstructured"
+                        jcr:title="LOGO AND ICONS"
+                        sling:resourceType="granite/ui/components/coral/foundation/container">
+                        <items jcr:primaryType="nt:unstructured">
+                            <logo
+                                jcr:primaryType="nt:unstructured"
+                                sling:resourceType="cq/gui/components/authoring/dialog/fileupload"
+                                fieldLabel="Upload logo here"
+                                fileReferenceParameter="./fileReference"
```
Comment (SwaroopCodilar) [inline ui.apps/src/main/content/jcr_root/apps/glucerna/components/footercopyRight/_cq_dialog/.content.xml]: provide root path

File: ui.apps/src/main/content/jcr_root/apps/glucerna/components/footercopyRight/_cq_dialog/.content.xml [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+                sling:resourceType="granite/ui/components/coral/foundation/tabs">
+                <items jcr:primaryType="nt:unstructured">
+                    <logoTab
+                        jcr:primaryType="nt:unstructured"
+                        jcr:title="LOGO AND ICONS"
+                        sling:resourceType="granite/ui/components/coral/foundation/container">
+                        <items jcr:primaryType="nt:unstructured">
+                            <logo
+                                jcr:primaryType="nt:unstructured"
+                                sling:resourceType="cq/gui/components/authoring/dialog/fileupload"
```
Comment (SwaroopCodilar) [inline ui.apps/src/main/content/jcr_root/apps/glucerna/components/footercopyRight/_cq_dialog/.content.xml]: use Path Browser

File: ui.apps/src/main/content/jcr_root/apps/glucerna/components/footercopyRight/footercopyRight.html (hunk 1 above)
Comment (IshappaN) [reply ui.apps/src/main/content/jcr_root/apps/glucerna/components/footercopyRight/footercopyRight.html]: Done

File: ui.apps/src/main/content/jcr_root/apps/glucerna/components/footercopyRight/_cq_dialog/.content.xml (hunk 2 above)
Comment (IshappaN) [reply ui.apps/src/main/content/jcr_root/apps/glucerna/components/footercopyRight/_cq_dialog/.content.xml]: Done

File: ui.apps/src/main/content/jcr_root/apps/glucerna/components/footercopyRight/_cq_dialog/.content.xml (hunk 3 above)
Comment (IshappaN) [reply ui.apps/src/main/content/jcr_root/apps/glucerna/components/footercopyRight/_cq_dialog/.content.xml]: Done

File: ui.apps/src/main/content/jcr_root/apps/glucerna/components/footercopyRight/_cq_dialog/.content.xml (hunk 4 above)
Comment (IshappaN) [reply ui.apps/src/main/content/jcr_root/apps/glucerna/components/footercopyRight/_cq_dialog/.content.xml]: Done

**Pull request merged**